src/openforms/forms/api/serializers/form.py

```diff
@@ -64,6 +65,9 @@ class FormSerializer(PublicFieldsSerializerMixin, serializers.ModelSerializer):
         default=list,
     )
     login_options = LoginOptionsReadOnlyField()
+    cosign_login_info = serializers.SerializerMethodField(
+        help_text=_("Info about the plugin to use for co-sign authentication."),
+    )
```

Comment:
It'd make a lot more sense to use the `LoginOptionSerializer` here instead of a `SerializerMethodField`. that will also result in the schema being correct.

You can use it as `cosign_login_info = LoginOptionSerializer(source="*")` to pass through the entire form instance, and then you have the request context available in that serializer via `self.context["request"]`.